Sming/Libraries/RCSwitch/RCSwitch.cpp

```diff
@@ -285,7 +285,7 @@ char* RCSwitch::getCodeWordB(int nAddressCode, int nChannelCode, boolean bStatus
    
    const char* code[5] = { "FFFF", "0FFF", "F0FF", "FF0F", "FFF0" };
    if (nAddressCode < 1 || nAddressCode > 4 || nChannelCode < 1 || nChannelCode > 4) {
-    return '\0';
+    return nullptr;
```

Comment:
@mikee47 What do you think about this change?

Reply:
Code will behave as it did before, which is probably wrong! It's used by `sendTriState()` which will then just segfault if it gets a NULL value. It should therefore be:

```
   if (nAddressCode < 1 || nAddressCode > 4 || nChannelCode < 1 || nChannelCode > 4) {
     sReturn[0] = '\0';
     return sReturn;
   }
```